### PR TITLE
Bing ads extract source doc

### PR DESCRIPTION
**Pull request by ishachopra0401**
## What do these changes do?

Created doc for Bing ads extract source

### CODE REVIEW

File: docs/sources/extract/bing-ads-v2.mdx [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+
+<div class="infoBlock">
+For more information on the schedule types, refer to the <Link to="/sources/extract/common-settings/">Common Settings</Link> guide.
+</div>
+
+### Selecting the data to import
+
+You can choose the Bing Ads data you want to ingest by selecting the required resources:
+
+<img src="../../assets/cloud-extract-sources/bing-ads-connection-settings-2.png" alt="Selecting the data to import" />
```
Comment (ishachopra0401) [inline docs/sources/extract/bing-ads-v2.mdx:57]: @nidhilashkari17 Need Oauth creds to add this image and verify the resources in this section

**Review by nidhilashkari17 — APPROVED** [review]
LGTM

File: docs/sources/extract/bing-ads-v2.mdx [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+
+1. Log into your [RudderStack dashboard](https://app.rudderstack.com/).
+2. Go to **Sources** > **New source** > **Cloud Extract** and select **Bing Ads V2** from the list of sources.
+3. Assign a name to your source and click **Continue**.
+
+### Connection settings
+
+Next, configure the following dashboard settings:
+
+<img src="../../assets/cloud-extract-sources/bing-ads-v2-settings.png" alt="Freshdesk credentials" />
```
Comment (ameypv-rudder) [inline docs/sources/extract/bing-ads-v2.mdx:30]:
```suggestion
<img src="../../assets/cloud-extract-sources/bing-ads-v2-settings.png" alt="Bing Ads credentials" />
```

File: docs/sources/extract/bing-ads-v2.mdx [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+You can choose the Bing Ads data you want to ingest by selecting the required resources:
+
+<img src="../../assets/cloud-extract-sources/bing-ads-connection-settings-2.png" alt="Selecting the data to import" />
+
+The below table mentions the syncs and primary keys supported by these resources from Bing Ads to your warehouse destination:
+
+| Resource | Sync type | Primary key |
+| :---| :--- | :----| 
+| `Account Performance Report` | Incremental | `AccountId`, `TimePeriod`, `CurrencyCode`, `AdDistribution`, `DeviceType`, `Network`, `DeliveredMatchType`, `DeviceOS`, `TopVsOther`, `BidMatchType` |
+| `Accounts` | Full refresh | `Id` |
```
Comment (ameypv-rudder) [inline docs/sources/extract/bing-ads-v2.mdx:62]:
```suggestion
| `Accounts` | Full Refresh | `Id` |
```

Comment (ameypv-rudder) [reply docs/sources/extract/bing-ads-v2.mdx:62]: To maintain consistency with the other docs. Can make this change throughout.

File: docs/sources/extract/bing-ads-v2.mdx [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
+2. Go to **Sources** > **New source** > **Cloud Extract** and select **Bing Ads V2** from the list of sources.
+3. Assign a name to your source and click **Continue**.
+
+### Connection settings
+
+Next, configure the following dashboard settings:
+
+<img src="../../assets/cloud-extract-sources/bing-ads-v2-settings.png" alt="Freshdesk credentials" />
+
+- **Tenant ID**: Enter the Tenant ID of your Microsoft Advertising developer application.
```
Comment (ameypv-rudder) [inline docs/sources/extract/bing-ads-v2.mdx:32]: We can add some info/reference link on how to obtain this ID.

**Review by ameypv-rudder — APPROVED** [review]

**Pull request merged**